### ISSUE TITLE
Unable to install using fish shell

### ISSUE THREAD

**Issue by gingerchris** [netlify/netlify-lm-plugin#28]
**- Do you want to request a *feature* or report a *bug*?**
bug

**- What is the current behavior?**
`netlify lm:install` fails with the following message:
```
Configuring Git to use Netlify's Git Credential Helper
Error: Unable to set credential helper in PATH. We don't how to set the path for fish shell.
Set the helper path in your environment PATH: /Users/user/.netlify/helper/bin
```

I have set the helper path in the path already; running `echo $PATH` gives me:
`/Users/chrisdowling/.netlify/helper/bin /Users/chrisdowling/.config/nvm/12.13.0/bin /usr/local/bin /usr/bin /bin /usr/sbin /sbin`

**- If the current behavior is a bug, please provide the steps to reproduce.**
Run `netlify lm:install` using fish shell.

**- What is the expected behavior?**
This should skip the part where it tries to set PATH, as the helper path is already set

**- Please mention your node.js, and operating system version.**
Node 12.13.0
OSX 10.14.6
Fish shell 3.0.2

**Comment by dannytce** [comment]
@gingerchris it looks like, it was already fixed in #26 (and reported in #24), but probably wasn't released yet. You can patch it by yourself in `/Users/dannytce/.local/share/netlify-cli/node_modules/netlify-lm-plugin/lib/install.js` 

Hope this helps :)

**Comment by gingerchris** [comment]
@dannytce thanks for this - I had seen the other issue but assumed as it was resolved this was a regression - I hadn't considered it not being released yet!
